### PR TITLE
[#42027] Improve 2FA Documentation

### PR DESCRIPTION
**Pull request by as-op**
https://community.openproject.org/work_packages/42027

Recreated PR https://github.com/opf/openproject/pull/11032

### CODE REVIEW

**Comment by ghost** [comment]
@as-op this pull request is based on opf
the old pull request was based on adam-op repo. i am not allowed to work on the opf repo,
you mentioned that you will close the old one and i will have to re edit the things, now i am unsure if the edits are inside this pull request or not.
please advice. what i need to re-edit or not, thank you

**Comment by as-op** [comment]
Sorry, yes, I could not create a new branch in the adam-op repository, so I used the core repo.
3 possible ways to re-edit:
* You can checkout "my" branch https://github.com/opf/openproject/tree/documentation/42027-improve-2fa-documentation into adam-op and work there (then you can close this PR, since you must recreate it)
* We merge it and you create an new branch at adam-op with the standard flow
* Or, since I don't see a good reason why you should not be allowed to exceptionally work directly in here. I recommend this way.

File: docs/installation-and-operations/configuration/README.md [hunk 1]
```diff
@@ -489,6 +489,15 @@ OPENPROJECT_2FA_SNS_SECRET__ACCESS__KEY="YOUR SECRET KEY"
 OPENPROJECT_2FA_SNS_REGION="AWS REGION"
 ```
 
+Further a REST integration could be used:
```
Comment (oliverguenther) [inline docs/installation-and-operations/configuration/README.md]: This should not be documented, it is an internal API used by a previous customer

Comment (oliverguenther) [reply docs/installation-and-operations/configuration/README.md]: I removed this in https://github.com/opf/openproject/commit/017e4364bb

**Review by oliverguenther — APPROVED** [review]

**Comment by ghost** [comment]
okay, thank you

**Pull request merged**